src/app/register/components/form-password/form-password.component.html

```diff
@@ -31,6 +31,12 @@
       >
         Password must match all pattern requirements
       </mat-error>
+      <mat-error
+        *ngIf="form.hasError('maxlength', 'password')"
+        i18n="@@register.passwordMaxLength"
+      >
+        Passwords must be 256 characters or less
```

Comment:
According to the designs this should read https://www.figma.com/file/dsrMiBCQsVngXCyjsoOLkw/Passwords-%26-validation?node-id=0%3A1

I have already added a tag for this `register.passwordIsToLongV2`

Reply:
```suggestion
        Password must be between 8 and 256 characters
```